### ISSUE TITLE
[Bug] `DrawingView` crashes if `Points` and/or `DrawingFinishedCommand` property is not set

### ISSUE THREAD

**Issue by jfversluis** [xamarin/XamarinCommunityToolkit#1451]
### Description

I tried to implement a simple `DrawingView`, but it crashed. Looking at the source I need to set both the `Points` and the `DrawingFinishedCommand` else it won't work.

### Steps to Reproduce

1.  Implement DrawingView like below

```xaml
<xct:DrawingView  BackgroundColor="LightGray" HeightRequest="200"  />
```

2. Run app
3. Try to draw, the app will crash when you take your finger/pointer of the canvas

### Expected Behavior

Line gets drawn

### Actual Behavior

NRE and application crashes

### Basic Information

- Version with issue: Current nightly
- Last known good version: N/A
- IDE: VSMac
- Platform Target Frameworks: 
  - iOS:  14.5
  - Android: Not test, but pretty sure it happens on Android as well
- Android Support Library Version: N/A

### Workaround
Assign values to `Points` and `DrawingFinishedCommand`

### Reproduction imagery

https://user-images.githubusercontent.com/939291/124086563-77ad8a00-da51-11eb-84d3-0410b99a43ef.mov

**Comment by jfversluis** [comment]
@VladislavAntonyuk FYI

**Comment by VladislavAntonyuk** [comment]
Not able to reproduce on Android. Will check on iOS later

**Comment by jfversluis** [comment]
Fixed in #1459
